app/helpers/stream_entries_helper.rb

```diff
@@ -53,11 +53,11 @@ def rtl_status?(status)
 
   def rtl?(text)
     text = simplified_text(text)
-    rtl_characters = /[\p{Hebrew}|\p{Arabic}|\p{Syriac}|\p{Thaana}|\p{Nko}]+/m.match(text)
```

Comment:
Yes.  Everything is okay. I said 'Oh...😅' because old code has '|' in '[]' in Regexp.